changelog/fragments/upgrade-operator-framework-api.yaml

```
@@ -0,0 +1,31 @@
+# entries is a list of entries to include in
+# release notes and/or the migration guide
+entries:
+  - description: >
+      Enabled definition of custom categories encoded as [JSON file(s)](https://olm.operatorframework.io/docs/tasks/validate-package/#validation)
+      for the `operatorhubio` validator.
```

I suggested the wrong validator name, my bad.
```suggestion
      for the `operatorhub` validator.
```